### PR TITLE
UI tests: Fix flakiness and added firmwareupdate spec

### PR DESCRIPTION
**Pull request by nabil-brn**
## 🦒 Context (issues, jira)

- `accounts.spec.ts` is failing on macOS
- `settings.spec.ts` rarely fails but it does, it's due to mouse moves and the scrollbar is showing up sometimes
- migrated `firmwareupdate.spec.ts`

## 🖤  Expectations to reach

CI must be green ✔️

### CODE REVIEW

**Review by valpinkman — DISMISSED** [review]

File: playwright/models/FirmwareUpdateModal.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  readonly readyCheckbox: Locator;
+  readonly downloadProgress: Locator;
+  readonly flashProgress: Locator;
+  readonly updateDone: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.readyCheckbox = page.locator('data-test-id=firmware-update-ready-checkbox');
+    this.downloadProgress = page.locator('data-test-id=firmware-update-download-progress');
```
Comment (ggilchrist-ledger) [inline playwright/models/FirmwareUpdateModal.ts:15]: Any need for these three?

File: playwright/models/Layout.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ export class Layout {
   constructor(page: Page) {
     this.page = page;
 
-    // portfolio
+    // portfolio && accounts
     this.totalBalance = page.locator('data-test-id=total-balance');
 
     // drawer
```
Comment (ggilchrist-ledger) [inline playwright/models/Layout.ts:30]: For future we should probably move Drawer into its own PO

File: src/renderer/modals/DisclaimerModal/index.js [hunk 3]
```diff
@@ -114,7 +114,7 @@ class DisclaimerModal extends PureComponent<Props, State> {
             <Box horizontal justifyContent="flex-end" alignItems="center" style={{ flex: 1 }}>
               <Box horizontal alignItems="center" onClick={this.onSeedReady} style={{ flex: 1 }}>
                 <CheckBox
-                  id={"firmware-update-disclaimer-modal-seed-ready-checkbox"}
```
Comment (ggilchrist-ledger) [inline src/renderer/modals/DisclaimerModal/index.js:117]: Are you sure this ID isn't getting used for anything else (like styling)?

Comment (valpinkman) [reply src/renderer/modals/DisclaimerModal/index.js:117]: as a convention, we try to not use IDs for styling, but nice catch anyway, we should indeed check it out (even tho i'm 99% sure we don't need to)

File: playwright/models/Layout.ts (hunk 2 above)
Comment (nabil-brn) [reply playwright/models/Layout.ts:30]: Agreed, we'll do it step by step 👍

File: playwright/models/FirmwareUpdateModal.ts (hunk 1 above)
Comment (nabil-brn) [reply playwright/models/FirmwareUpdateModal.ts:15]: They're used by po itself and spec file too.

File: src/renderer/modals/DisclaimerModal/index.js (hunk 3 above)
Comment (nabil-brn) [reply src/renderer/modals/DisclaimerModal/index.js:117]: Sure, I looked for the id in the whole code.

**Review by ggilchrist-ledger — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]

**Pull request merged**